cpp/src/io/json/read_json.cu

```diff
@@ -72,7 +72,6 @@ device_span<char> ingest_raw_input(device_span<char> buffer,
   // line of file i+1 don't end up on the same JSON line, if file i does not already end with a line
   // delimiter.
   auto constexpr num_delimiter_chars = 1;
```

Comment:
Maybe there should be a comment on ingest_raw_input that the returned span can be larger than range_size when reading multiple sources